src/mmw/js/src/data_catalog/templates/cuahsiSearchResult.html

```diff
@@ -1,22 +1,19 @@
 <h3 class="resource-title">
-    {{ id }}
+    <a href="{{service_url}}" target="_blank">{{ service_code }}</a> {{ id | split(":", 1) }}
```

Comment:
While we're here, could you try and restore the `details_url` link to the second part of this title? It was lost in 81d18ae. Basically, if we have a `details_url` we should wrap `{{ id | split(":", 1) }}` in it, else leave it as is.